examples/factors/.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
```

Comment:
I don't think this is necessary, we have a top-level `.gitignore` which should do that right thing

Reply:
replaced with a `.gitignore` that expects the `example` lock files